docs/common/cursor_c.ldoc

```diff
@@ -43,6 +43,7 @@
  * <div>man</div>
  * <div>middlebutton</div>
  * <div>mouse</div>
+ * <div>nil</div>
```

Comment:
The rest of the list are string values (e.g. the documented `mouse` actually means "pass `"mouse"`"), and `"nil"` is not correct.
And because `nil` is a special case here, I'd say that it should instead be explained in the prose part of the function's documentation.